presto-geospatial/src/main/java/com/facebook/presto/plugin/geospatial/SphericalGeoFunctions.java

```diff
@@ -105,6 +115,17 @@ public static Slice toGeometry(@SqlType(SPHERICAL_GEOGRAPHY_TYPE_NAME) Slice inp
         return input;
     }
 
+    @Description("Returns the Well-Known Text (WKT) representation of the geometry")
+    @ScalarFunction("ST_AsText")
+    @SqlType(VARCHAR)
+    public static Slice stSphericalAsText(@SqlType(SPHERICAL_GEOGRAPHY_TYPE_NAME) Slice input)
+    {
+        if (input == null) {
+            return utf8Slice("POINT EMPTY");
```

Comment:
If the input is null, the output should be null.  ScalarFunctions do this automatically, unless you use the `@SqlNullable` annotation.

Reply:
Removed it